scripts/make_addon_table.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+gather_data(){
+    for addon in addons/*/*.yaml; do
+        KIND=$(yq -r '.kind' "${addon}")
```

Comment:
seems `yq` on linux vs mac are different. maybe wrap this in a container?

Reply:
yq is not shipped with the OS, is it? Since I need this for the release, I'm going to merge as-is. It will work if the correct yq is installed and will work in CI.

Happy to accept improvements in a future PR.